package.json

```diff
@@ -7,7 +7,7 @@
   "license": "N/A",
   "private": true,
   "scripts": {
-    "dev": "cross-env TESTING_MODE=true APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362 GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda SUBMIT_FEEDBACK_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitFeedback NODE_OPTIONS=--inspect next",
```

Comment:
Why did you remove the `--inspect` that allowed us to do breakpoint debugging in vscode?

Reply:
Since the version updates, something is wrong here.

If you add inspect how it's described in the Next CLI documentation you get the following error:
```Error: Use env variable NODE_OPTIONS instead: NODE_OPTIONS="--inspect" next dev```

If you follow the fix instructions, it works, but the next server will randomly crash while you're developing. It's very frustrating. I looked into the issue and it's to do with the fact that multiple node debuggers are trying to launch (but only sometimes). I think the option is from node and not Next.

My hunch is that between cross-env and next and version updates something has changed and it's not working. I imagine if I ran next without cross-env it would be ok, but we need those environment variables. Decided to remove because the crashing was the worst scenario but would like to return the functionality.